cli-feed-v4.json

```diff
@@ -11958,6 +11958,241 @@
           "default": "false"
         }
       ]
+    },
+    "4.31.0": {
```

Comment:
I went with a minor version rev as I think new templates qualify as a new "feature", thus following semver this is a minor version rev.

Reply:
Yeah that works. The only thing i would say is co-ordinate with the release team if they are doing any updates. As their changes might end up overwriting this release.

Reply:
Who should I reach out to on the release team?